Library: document new text filter operator '=' (find exact matches) (#590)
mixxxdj/manual: source/chapters/library.rst

     It doesn't matter if you have space between the colon and the argument
     or not. Quotes must be used for multi-word text arguments.

>   You can use the **=** operator to find only tracks where the entire property

Reviewer: ```suggestion
  You can use **=** for finding exact matches. It reveals only tracks where the entire property
```

>   You can use the **=** operator to find only tracks where the entire property
>   text equals the search term. You can also enclose multiple terms in quotes.

Reviewer 2: I think this is clearer.
```suggestion
  You can use the **=** operator to find only tracks where the entire property
  text equals the search term. As without **=**, you must use quotes for more than one word.
```

  The first of the examples below will find only tracks where the title is

>   "wow" (case-insensitive like the default search), i.e. not "wow!", "wowy"
>   or "wow (Hej Remix)".

Reviewer 2: I think there should also be an example with a prefix, eg: 
```suggestion
  "wow" (case-insensitive like the default search), i.e. not "wowy", "oh wow!"
  or "wow (Hej Remix)".
```
